lib/importer/sequence_ontology_mirror.rb

```diff
@@ -33,8 +35,12 @@ def populate_ontology_entry
     end
 
     def valid_entry?(entry)
-      ['id', 'name'].inject(true) do |val, term|
-        entry[term].present? && val
+      if entry['is_obsolete'].present?
+        obsolete_terms.append(entry)
```

Comment:
`append` will return a copy of the array, which will be a truthy value, which means `valid_entry?` will report as true and the term will be added, only to be removed when we `process_obsolete_terms`. Is that intentional? Or would it make sense to return something false here and not add them in the first place if they're not already present?

Reply:
I didn't realize it would return a truthy value. I added a line to `return false`